src/deepsparse/transformers/helpers.py

```diff
@@ -193,3 +199,93 @@ def _normalize_fields(_dict):
         return result
 
     return _wrapper
+
+
+def truncate_transformer_onnx_model(
+    model_path: str,
+    emb_extraction_layer: int = -1,
+    model_size: Optional[int] = None,
+    final_node_name: Optional[str] = None,
```

Comment:
to avoid hidden errors if the user is using a custom model (ie something we might not support yet like `roberta`) and also to avoid needing to constantly update the layer inference when models are added, let's make this a required variable that the user is to provide to the pipeline and document.

Reply:
Maybe optional with a warning? As a user of the pipeline, this argument makes it very easy to quickly test different layers of embeddings without having to manually find the last bert layer within the onnx graph.

Reply:
It's also supported by Haystack, so it would be nice to support it here as well.